src/components/Search.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+class Search extends React.Component {
+    constructor() {
+        super();
+
+        this.state= {
+            query: ''
+        }
+
+        this.handleChange = this.handleChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    handleChange(event) {
+        this.setState({
+            query: event.target.value
+        })
+    }
+
+    handleSubmit(event) {
+        event.preventDefault();
+        this.props.receiveQuery(this.state.query);
+        document.querySelector(".search__input").value = "";
+    }
+
+    render() {
+        return (
+            <form className={this.props.className} onSubmit={this.handleSubmit}>
```

Comment:
it looks like the classname gets set on the parent and used again here. It might make more sense to set the className directly on this element rather than pass it down as a prop